pkg/controller/siddhiprocess/config.go

```diff
@@ -423,16 +423,20 @@ func (rsp *ReconcileSiddhiProcess) Configurations(sp *siddhiv1alpha2.SiddhiProce
 	configMap := &corev1.ConfigMap{}
 	err := rsp.client.Get(context.TODO(), types.NamespacedName{Name: cmName, Namespace: sp.Namespace}, configMap)
 	if err == nil {
-		if configMap.Data["siddhiRunnerHome"] != "" {
-			configs.SiddhiHome = configMap.Data["siddhiRunnerHome"]
+		if configMap.Data["siddhiHome"] != "" {
```

Comment:
You can use the 2nd value return by the map to check whether the key is available or not.
```go
if v, ok := configMap.Data["siddhiHome"] ; ok {
...
}
```

Reply:
Fixed in https://github.com/siddhi-io/siddhi-operator/pull/123/commits/5a2fe203135ca70441588c95b228e415d49f3f8a